Blocks/SS/idx_ts.h

```diff
@@ -59,6 +60,9 @@ CURRENT_STRUCT(TimestampAndOptionalIndex) {
   CURRENT_CONSTRUCTOR(TimestampAndOptionalIndex)(std::chrono::microseconds us, uint64_t index) : us(us), index(index) {}
 };
 
+// The `head` is either the timestamp of the last published entry or of the last `UpdateHead()` call,
+// depending on whether the last entry was published after the last `UpdateHead()` call or not.
```

Comment:
Why not `depending on whether the most recent update was '.Publish()' or '.UpdateHead()'`?